invertedai/utils.py

```diff
@@ -1144,16 +1144,30 @@ def _update_agent(self, agent_idx, agent, agent_attribute):
 
         box = np.matmul(rot(psi), box.T).T + np.array([[x, y]])
         if self.direction_vec:
+            marker_offset = agent_attribute.length/4
+            x_data = x + marker_offset*math.cos(psi)
+            y_data = y + marker_offset*math.sin(psi)
+            marker_data = (3, 0, (-90+180*psi/math.pi))
+
             if agent_idx not in self.dir_lines:
+                # self.dir_lines[agent_idx] = self.current_ax.plot(
```

Comment:
What are these commented code for ? Should it be removed ?

Reply:
Fixed